ui/apps/platform/src/Containers/NetworkGraph/simulation/NetworkPoliciesYAML.tsx

```diff
@@ -53,17 +63,15 @@ function NetworkPoliciesYAML({ yaml, generateNetworkPolicies }: NetworkPoliciesY
         />
     ) : null;
 
-    if (!yaml || yaml === '') {
-        return (
-            <Bullseye>
-                <EmptyState variant={EmptyStateVariant.xs}>
-                    <Title headingLevel="h4" size="md">
-                        No network policies
-                    </Title>
-                </EmptyState>
-            </Bullseye>
-        );
-    }
+    const revertRecentYAML = undoNetworkPolicies ? (
```

Comment:
Why is this optional?

Reply:
Hmm, actually that's a good question. I think the controls should be visible regardless and not hidden when viewing active YAMLs. I'll change that